packages/components/src/components/Address/Address.stories.tsx

```diff
@@ -9,15 +9,22 @@ export default {
   component: Address,
 };
 
-export const Default = () => <Address value='testaddress.eth' />;
+export const Default = () => <Address value='testaddress.eth' shortened={false} />;
 
-const AddressUsingProvider = () => {
+export const DefaultShortenedWithENS = () => <Address value='testaddress.eth' shortened={true} />;
```

Comment:
true is implied when you set a boolean prop on a component in this way

```suggestion
export const DefaultShortenedWithENS = () => <Address shortened value='testaddress.eth' />;
```